stanford_soe_regions.module

```diff
@@ -137,6 +137,10 @@ function stanford_soe_regions_theme() {
       'template' => 'templates/digital-magazine-page',
       'render element' => 'page'
     ),
+    'print__html' =>  array(
```

Comment:
This solution looks good but I have a couple of questions to understand the issue and resolution path.

Why have you added this theme hook here? Where did it go in print 2.2? Was this ever an option? 
What I want to know is why it worked before and why you chose to fix this issue by adding it to this module.

Reply:
I added it there because I didn't want to put it in the theme and the regions module is where the digital magazine template is located, so it seemed proper. 

As far as why it worked before, I'm not 100% sure on that, but it had something to do with this: https://www.drupal.org/node/2276407

I figured I could get it working in 2.2 and then get a template to load correctly.